src/fetch_package.js

```diff
@@ -12,15 +12,21 @@
  * file, to avoid letting this server execute arbitrary code.
  */
 
+import request from "superagent";
 import profile from "./profile.js";
 
-const request = require("superagent");
+import type {JavaScriptPackage, RequestStats} from "./types.js";
+
+type InflightRequests = {
+    [url: string]: Promise<JavaScriptPackage>,
+    ...,
+};
 
 // How many times we retry on 5xx error or similar, before giving up.
-const numRetries = 2; // so 3 tries total
+const DefaultNumRetries: number = 2; // so 3 tries total
```

Comment:
Hmm! When I see something LikeThis I assume it's either a class, something imported, or a type - not a constant. I think if we want this to just be a constant then doing `const DEFAULT_NUM_RETRIES: number = 2;` is fine.

Reply:
Sorry, carryover from my C# days. I'll change it.